platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -566,6 +572,20 @@ public MapboxMapOptions setPrefetchesTiles(boolean enable) {
     return this;
   }
 
+  /**
+   * Enable cross source collision, defaults to true.
+   * <p>
+   * If set to false, it automatically groups layers based on their source ID.
```

Comment:
How about something like "if set to false, symbol layers will only run collision detection against other symbol layers that are part of the same source"?